backend/typescript/models/application.model.ts

```diff
@@ -0,0 +1,37 @@
+import { Column, DataType, Model, Table } from "sequelize-typescript";
+
+@Table({ tableName: "applications" })
+export default class Application extends Model {
```

Comment:
It seems this model is missing a few fields like the first choice, second choice, resume link, etc. Is that not stored in the firebase application? These are fields our frontend application would need so it is very important to get them or have another request to get them.